_posts/2023-07-19-trino-fest-2023-stripe.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Inspecting Trino on ice"
+author: "Kevin Liu, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Stripe.png
+show_pagenav: false
+---
+
+Stripe is an online payment processor that facilitates online payments for
+digital native merchants. They use Trino to facilitate ad hoc analysis, enable
+dashboarding, and provide internal API in order for services to utilize Trino to
+query information. In Kevin Liu's session at [Trino Fest 2023]({% post_url
+2023-06-20-trino-fest-2023-recap %}), he details the power of Trino connectors
+and how it can even replace some functionalities. He also discusses how Trino
+is at the core of all operations at Stripe.
+
+<!--more-->
+
+{% youtube PSGuAMVc6-w %}
+
+## Recap
+
+Trino is the foundational infrastructure on which other apps and
+services are built upon. In Kevin's words, "I call Trino the Swiss army knife in
+the data ecosystem." At Stripe, they use Iceberg tables extensively and want to
+migrate away from Hive. Some problems with the Iceberg CLI is that it requires
+an internal developer machine that accesses their data, which is painful to use
+for them. The output is also in JSON format, which is not human readable.
+However using the Trino Iceberg connector, it can replace some of the
+functionalities of the Iceberg CLI.
+
+Unfortunately, there was no way to grant table property information from the
```

Comment:
Maybe 

Since Stripe is not using one of the latest versions of Trino, not all metadata about the Iceberg tables properties is available directly in the catalog. However they decided to create a catalog that uses the PostgreSQL connector and is configured to access the Hive metastore table. It contains that missing information. With this setup, they have all the information...